server/routes/settings/index.ts

```diff
@@ -372,6 +374,23 @@ settingsRoutes.get(
       'data',
     ];
 
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const deepValues = (obj: Record<string, any>): any[] => {
```

Comment:
```suggestion
    const deepValues = (obj: Record<string, unknown>): unknown[] => {
```

Maybe `unknown` type here is better?

Reply:
Also, instead of looping through the whole object this way, I wonder if it would be faster to just JSON.stringify the entire log and just test against that string instead?

Reply:
> Also, instead of looping through the whole object this way, I wonder if it would be faster to just JSON.stringify the entire log and just test against that string instead?

Trying this out with a log file of 10k lines, it is slightly faster (an average of 5.2ms vs 5.9ms over a dozen tries) to search through all of it, but it also is less accurate since the log message keys (`timestamp`, `label`, etc) themselves are also being searched through.

> Maybe unknown type here is better?

If I set the type as unknown, I have to then assert the type when calling the function, so I opted for having it set as `any` instead in the function definition itself (still not ideal 😕).